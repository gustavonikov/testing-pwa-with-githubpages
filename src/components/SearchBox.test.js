import { shallow } from 'enzyme';
import SearchBox from './SearchBox';

it('renders SearchBox component', () => {
  expect(shallow(<SearchBox />)).toMatchSnapshot()
})
