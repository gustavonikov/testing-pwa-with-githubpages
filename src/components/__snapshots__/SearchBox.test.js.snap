// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchBox component 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search Robots"
    className="pa3 ba b--green bg-lightest-blue search-input"
    placeholder="search robots"
    type="search"
  />
</div>
`;
